packages/workspace/src/helpers/utils.ts

```diff
@@ -1,14 +1,31 @@
 import { ConfigurationService, ConfigurationServiceHttp } from '@capsulajs/capsulajs-configuration-service';
 import { API } from '..';
 import * as INTERNAL_TYPES from './types';
-import { bootstrapComponentError, bootstrapServiceError } from './const';
+import {
+  getLoadingServiceError,
+  getLoadingComponentError,
+  getBootstrapServiceError,
+  getInitComponentError,
+  getBootstrapComponentError,
+} from './const';
```

Comment:
This shouldn't be on some `errors.ts` file just to be more clear ?

Reply:
I was thinking about it, but errors are just conts - seems enough to me